test/regress.failures

```diff
@@ -4,3 +4,22 @@ test/regress/ext:gc/array_new_fixed.bin.wast
 test/regress/ext:gc/br_on_non2.bin.wast
 test/regress/ext:gc/br_on_non.bin.wast
 test/regress/ext:gc/ref_eq.bin.wast
+test/regress/ext:threads/atomic_load_i32.bin.wast
```

Comment:
Do these tests only fail on the single-pass compiler now? If so, the failures file should be at least platform-specific, since these should all pass on the V3 interpreter.

Reply:
Oh, it passes on V3 interpreter. I was planning to remove all of these when making a final PR to V3 interpreter.

Reply:
Ok, sgtm.